CMakeLists.txt

```diff
@@ -623,6 +623,11 @@ if(CMAKE_C_COMPILER_ID MATCHES Clang)
   add_compile_options($<$<COMPILE_LANGUAGE:CXX>:-Werror=c++98-compat-extra-semi>)
 endif()
 
+# Make sure we know where swift-syntax is because we need it to build the parser.
+if(NOT EXISTS "${SWIFT_PATH_TO_SWIFT_SYNTAX_SOURCE}")
```

Comment:
We should probably stick to a more conventional directory naming scheme.
Something like `SWIFT_SYNTAX_SOURCE_DIR`.

Reply:
I think that the name actually works well for the current state of things - all the external repositories use this current naming scheme.

Reply:
This is true. Diverges from LLVM, but I guess that's common around these parts.